python_scripts/AddressPoint.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python2.7
+# -*- coding: utf-8 -*-
+#
+#Short script for formating AddressPoint datafile
+#run using python AddressPoint.py -i <filnamn>
+#
+import codecs, sys, getopt
+
+def main(argv):
+    inputfile = ''
+    outputfile = ''
+    try:
+        opts, args = getopt.getopt(argv,"hi:")
+    except getopt.GetoptError:
+        print u'python AddressPoint.py -i <filnamn>'
+        sys.exit(2)
+    for opt, arg in opts:
+        if opt == '-h':
+            print u'python AddressPoint.py -i <filnamn>'
+            sys.exit()
+        elif opt == "-i":
+            inputfile = arg
+    if len(inputfile) > 0:
+        outputfile = u'%s-mod.txt' %inputfile[:-4]
+        try:
+            fIn=codecs.open(inputfile,'r','Latin-1')
+        except IOError:
+            print u'Skrev du verkligen rätt filnamn?'
+            sys.exit(2)
+        lines = fIn.read().split(u'\n')
+        fIn.close()
+        
+        fOut=codecs.open(outputfile,'w','utf8')
+        for l in lines:
```

Comment:
PEP 8 recommends [not using `l` as variable name](https://www.python.org/dev/peps/pep-0008/#id38).